values.yaml

```diff
@@ -91,8 +91,8 @@ kpi:
     secret:
       {}
       # DATABASE_URL: "postgres://postgres:password@postgres-postgresql:5432/postgres"
-      # CELERY_BROKER_URL: "redis://localhost:6379/2"
-      # CACHE_URL: "redis://localhost:6379/4"
+      # CELERY_BROKER_URL: "redis://localhost:6379/1"
```

Comment:
Could this create zombie tasks? (e.g. workers and broker not speaking to each other until the deployment is totally done?)